src/Bridges/ApplicationLatte/TemplateFactory.php

```diff
@@ -48,7 +54,11 @@ public function __construct(ILatteFactory $latteFactory, Nette\Http\IRequest $ht
 	public function createTemplate(UI\Control $control = NULL)
 	{
 		$latte = $this->latteFactory->create();
-		$template = new Template($latte);
+		$template = new $this->templateClassName($latte);
```

Comment:
Ok, let's skip it now then.

Btw, do you need `$latte` as argument in your implementation?